inscriptions/ISic020196.xml

```diff
@@ -92,8 +92,11 @@
                             <handNote>
                                 <!--  Based on Glyph table: eta02 or omicron07; omicron04. A vertical bar near the letters (numeral 1?) -->
                                 <locus from="line1" to="line1">Line 1</locus>
-                                <dimensions type="letterHeight">
-                                    <height unit="mm"/>
+                                <dimensions xml:id="t1" type="letterHeight">
```

Comment:
and i think here some sort of explanation is needed, at least in comment, since the xml ids are opaque to me, but perhaps are meant to refer to each of the individual symbols/characters?